main/convert.js

```diff
@@ -220,6 +236,11 @@ const convertToGif = PCancelable.fn(async (options, onCancel) => {
     ),
     options.outputPath
   ]);
+
+  return compress(options.outputPath, options, [
```

Comment:
Are we running the compression at all times? And the setting only changes whether it's lossy or not?

Reply:
Yes, that's correct. The gifs were always so large so I thought this might be beneficial.